lib/tools/apk-utils.js

```diff
@@ -152,7 +164,8 @@ apkUtilsMethods.startApp = async function startApp (startAppOptions = {}) {
     }
     return stdout;
   } catch (e) {
-    throw new Error(`Cannot start the '${startAppOptions.pkg}' application. ` +
+    const appDescriptor = startAppOptions.pkg || `${startAppOptions.action} ${startAppOptions.optionalIntentArguments}`;
```

Comment:
what is startAppOptions.optionalIntentArguments is unset?

Reply:
I just missed this one first time round trying to get the CI tests to pass! Fixed this in last commit, along with the refactor above to reuse `intentName`